src/server/autocomplete.d

```diff
@@ -801,9 +801,20 @@ DSymbol*[] getSymbolsByTokenChain(T)(Scope* completionScope,
 
 	if (shouldSwapWithType(completionType, symbols[0].kind, 0, tokens.length - 1))
 	{
-		symbols = symbols[0].type is null || symbols[0].type is symbols[0] ? [] : [symbols[0].type];
 		if (symbols.length == 0)
 			return [];
+        if (symbols[0].type is null || symbols[0].type is symbols[0])
+        {
+            symbols = [];
```

Comment:
I am bad at code reviews. You put the modifications to the symbols variable *after* the
```d
if (symbols.length == 0)
    return [];
```
check and I didn't catch it.

https://github.com/Hackerpilot/DCD/issues/341